tests/std/tests/VSO_0000000_vector_algorithms/test.cpp

```diff
@@ -922,6 +978,16 @@ void test_vector_algorithms(mt19937_64& gen) {
     test_reverse_copy<double>(gen);
     test_reverse_copy<long double>(gen);
 
+    test_remove<char>(gen);
+    test_remove<signed char>(gen);
+    test_remove<unsigned char>(gen);
+    test_remove<short>(gen);
+    test_remove<unsigned short>(gen);
+    test_remove<int>(gen);
+    test_remove<unsigned int>(gen);
+    test_remove<long long>(gen);
+    test_remove<unsigned long long>(gen);
+
     test_swap_ranges<char>(gen);

```

Comment:
It looks like the definitions aren't being added in the same order as their calls. When possible, it's good to follow the same order, making it easier to verify that all test code is actually being called.

Reply:
Swapped the definitions